riak-driver/riak-key-value/riak-key-value-api/src/main/java/org/jnosql/diana/riak/key/RiakKeyValueEntityManagerFactory.java

```diff
@@ -0,0 +1,59 @@
+package org.jnosql.diana.riak.key;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Queue;
+import java.util.Set;
+
+import org.jnosql.diana.api.key.BucketManagerFactory;
+
+import com.basho.riak.client.api.RiakClient;
+import com.basho.riak.client.core.RiakCluster;
+import com.basho.riak.client.core.query.Namespace;
+import com.google.gson.Gson;
+
+public class RiakKeyValueEntityManagerFactory implements BucketManagerFactory<RiakKeyValueEntityManager> {
+
+	RiakCluster cluster;
+	
+	@Override
+	public RiakKeyValueEntityManager getBucketManager(String bucketName) throws UnsupportedOperationException {
+		
+		cluster.start();
+		RiakClient riakClient = new RiakClient(cluster);
+		Namespace quotesBucket = new Namespace(bucketName);
+
+		return new RiakKeyValueEntityManager(riakClient, new Gson(),quotesBucket);
+	}
+
+	@Override
+	public <T> List<T> getList(String bucketName, Class<T> clazz) throws UnsupportedOperationException {
```

Comment:
Does Riak can support this method? 
If not, just throw an exception: 
`throw new UnsupportedOperationException("There is not support to getList on Riack");`